src/calendar/CalendarUpdateDistributor.js

```diff
@@ -72,6 +73,21 @@ export class CalendarMailDistributor implements CalendarUpdateDistributor {
 			body: makeInviteEmailBody(sender, event, message),
 			event,
 			sender
+		}).catch(RecipientsNotFoundError, e => {
+			// we want to delete the event even if the recipient is not an existing tutanota address
```

Comment:
is this comment correct? i'm not sure it matches what im seeing

Reply:
I think so. This refers to the following case: the user wants to delete a calendar event but there are invalid (attendee) addresses in it. usually we send out a mail that the event was cancelled to all attendees, but in this case we can't and receive an error. so we can't delete the event. therefore, i decided, it was best to ignore invalid recipients and only send the notifications to those recipients that we can resolve the mail address for. i thought, it would be to annoying to bother the user to manually remove or correct these mail addresses before being able to delete an event. we can discuss it, though.
anyway, prior to this there was already another similar case: external attendees without a password of a confidential event also will not receive a notification when we delete the event.

Reply:
I will drop the last commit